integration-tests/test/rnode.py

```diff
@@ -251,11 +251,11 @@ def eval(self, rho_file_path: str) -> str:
         return self.rnode_command('eval', rho_file_path)
 
     def deploy(self, rho_file_path: str, private_key: PrivateKey, phlo_limit:int = DEFAULT_PHLO_LIMIT,
-               phlo_price: int = DEFAULT_PHLO_PRICE, valid_after_block_no:int=0) -> str:
+               phlo_price: int = DEFAULT_PHLO_PRICE, valid_after_block_no:int=0, shard_id: str = '') -> str:
```

Comment:
We should define some global constant for default shardId used in tests and just override when needed.

Reply:
Done in [3b3c313](https://github.com/rchain/rchain/pull/3631/commits/3b3c313b9b064892f0463cbbaa89c454408ae07e) and [bb228f4](https://github.com/rchain/rchain/pull/3631/commits/bb228f475fdde33ccf63aba97b72f10096906322).